runtime/makelib/mkconstants.mk.ftl

```diff
@@ -24,9 +24,17 @@
 </#list>
 
 # Define the Java Version we are compiling
-VERSION_MAJOR?=9
+ifndef VERSION_MAJOR
+$(error VERSION_MAJOR is not set from extensions code)
+endif
 export VERSION_MAJOR
 
+# Define full Java Version
+ifndef VERSION_NUMBER_FOUR_POSITIONS
```

Comment:
I think this name should be more explicit since there are many parts and versions.
i.e. OPENJDK_VERSION_NUMBER_FOUR_POSITIONS

Reply:
I've updated the name to this.